{"name": "test_invalid_data[data6]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 210 == 400\n  +210\n  -400", "trace": "self = <test_api_creation.TestAPICreation object at 0x112fea0d0>, api_client = <ApiClient(Thread-24, initial)>\ndata = {'email': 'small_length_username@mail.ru', 'password': 'small_length_username_pas', 'username': 's'}\n\n    @pytest.mark.API\n    @pytest.mark.parametrize(\"data\", [data11, data21, data31, data41, data51, data61, data71])\n    def test_invalid_data(self, api_client, data):\n    \n        response = api_client.create(**data)\n>       assert response.status_code == 400\nE       assert 210 == 400\nE         +210\nE         -400\n\napi_client = <ApiClient(Thread-24, initial)>\ndata       = {'email': 'small_length_username@mail.ru', 'password': 'small_length_username_pas', 'username': 's'}\nresponse   = <Response [210]>\nself       = <test_api_creation.TestAPICreation object at 0x112fea0d0>\n\ntests/tests_api/test_api_creation.py:56: AssertionError"}, "parameters": [{"name": "data", "value": "{'username': 's', 'password': 'small_length_username_pas', 'email': 'small_length_username@mail.ru'}"}], "start": 1589971078521, "stop": 1589971078549, "uuid": "43e47def-8afd-48c1-9645-b110cb2786fb", "historyId": "28fa2f40abb76c6f47d352234bca3c29", "testCaseId": "0376a905d4248e62a552bb387e2a6e5e", "fullName": "tests.tests_api.test_api_creation.TestAPICreation#test_invalid_data", "labels": [{"name": "tag", "value": "API"}, {"name": "parentSuite", "value": "tests.tests_api"}, {"name": "suite", "value": "test_api_creation"}, {"name": "subSuite", "value": "TestAPICreation"}, {"name": "host", "value": "MacBook-Air-Mac.local"}, {"name": "thread", "value": "2637-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_api.test_api_creation"}]}